ArduCopter/Parameters.cpp

```diff
@@ -188,6 +188,15 @@ const AP_Param::Info Copter::var_info[] = {
     // @User: Standard
     GSCALAR(wp_yaw_behavior,  "WP_YAW_BEHAVIOR",    WP_YAW_BEHAVIOR_DEFAULT),
 
+    // @Param: LAND_ACCEL
+    // @DisplayName: Land acceleration 
+    // @Description: Limits maximum acceleration during landing. WPNAV_ACCEL is used if this parameter is zero
+    // @Units: cm/s/s
+    // @Range: 0 100
+    // @Increment: 1
+    // @User: Advanced
+    GSCALAR(land_accel_limit, "LAND_ACCEL", 0),
```

Comment:
i think it would make more sense to be WPNAV_LAND_ACCEL

Reply:
Right , I will update it now.